internal-distro.py

```diff
@@ -76,6 +76,7 @@ def main():
     parser.add_argument('distribution_file', type=str)
     parser.add_argument('--package', type=str, action='append', dest='target_packages', default=[])
     parser.add_argument('--repository', type=str, action='append', dest='target_repositories', default=[])
+    parser.add_argument('--manifest_file', type=str, default='package.xml')
```

Comment:
Nit: Can we make this `--manifest-file` instead? This doesn't change the code below (it'd still be `args.manifest_file`).